### DISCUSSION TITLE
Time for 1.0.19 release?

### DISCUSSION THREAD

**Discussion by tmshort** [jedisct1/libsodium#1136] [Q&A]
1.0.18 was tagged over 30 months ago (ok, the commit was made over 30 months ago). Does it make sense to create a 1.0.19 release, given the number of changes that have occurred since then?

**Comment by cmonfortep** [comment]
I'm also interested in understanding if the repo stopped using tags for new releases, or if 1.0.18 is actually the most recent stable version.

**Reply by MartyIX** [reply]
(There are some 1.0.18.* versions mentioned here: https://www.nuget.org/packages/libsodium#versions-tab but I don't know an answer to your question.)

**Reply by samuel-lucas6** [reply]
@MartyIX The NuGet versioning is for [updated platform support](https://github.com/jedisct1/libsodium/issues/1075#issuecomment-1080020946) rather than using the latest stable release like below. For example, it's not possible to use AEGIS.

**Comment by Skycoder42** [comment]
As far as I can tell, important changes are also pushed to the `stable` branch - and the `*-stable*` tarballs and compiled binaries are updated here: https://download.libsodium.org/libsodium/releases/

As you can see there, the last update to the 1.0.18 stable was done on about a week ago:
```
libsodium-1.0.18-stable.tar.gz                     01-Mar-2023 16:05             1874068
```

**Comment by vt-alt** [comment]
Aren't there is too much for important changes in the stable branch? It is actually unknown what is meaning of stable branch there and how important these changes, Does it promise not to beak API/ABI?

Should distributions update their libsodium packages from the stable branch ASAP or not? Without this clarifications package maintainers only update on release tags or on CVE published.

Currently is almost no package is updated beyond 1.0.18 https://repology.org/project/libsodium/versions Which is 4 years old.

**Reply by MartyIX** [reply]
cc @jedisct1

**Comment by jedisct1** [comment]
1.0.19 has been released.

The `stable` now apply to 1.0.19.

No API/ABI breakage is expected in `stable` releases. And if there's a security issue, a point release will follow shortly. So, following `stable` is safe, and distributions can do so in order to get occasional performance improvements, or to fix compilation issues on new platforms or with new toolchains.

**Reply by vt-alt** [reply]
This is great to know. Thanks!
